dds/DCPS/transport/rtps_udp/RtpsUdpDataLink.cpp

```diff
@@ -1670,6 +1672,8 @@ RtpsUdpDataLink::RtpsReader::process_data_i(const RTPS::DataSubmessage& data,
     return false;
   }
 
+  const GuidConverter conv(src);
```

Comment:
Is this used anywhere?

Reply:
Hmm, looks like a debugging remnant. Good catch!